spidermon/templates.py

```diff
@@ -71,6 +71,11 @@ def reload_env(self):
             self.env.globals[global_name] = global_value
 
     def get_template(self, name):
+        if os.path.isabs(name):
+            self.paths.append(os.path.dirname(name))
```

Comment:
why not `self.add_path`?

Reply:
Because I didn't notice that it already exists :-) I will change that as well.